src/main/java/seedu/uninurse/logic/commands/PatientsToday.java

```diff
@@ -0,0 +1,25 @@
+package seedu.uninurse.logic.commands;
```

Comment:
Is it better if this file is named PatientsTodayCommand like the rest?

Reply:
oops, a lot of naming got messed up when I merged the Uninurse naming PR, let me refactor them again